src/domain/books/BooksPage.tsx

```diff
@@ -1,7 +1,45 @@
+import { useDispatch, useSelector } from "react-redux";
+import React, { useEffect, useState } from "react";
+import { cleanupBooks, getBooks } from "./booksSlice";
+
+import { AppDispatch, RootState } from "../../store/store";
+import { Book } from "./types";
+import { BookCard } from "../../components/BookCard";
+import { addBookItemsTocart } from "../cart/cartSlice";
 export const BooksPage = (): JSX.Element => {
+  const { books: booksData } = useSelector((state: RootState) => state.books);
```

Comment:
Specify your selectors inside the slice file and use here.
`const { books } = useSelector(booksSelector);`